changelog/0.1.0.md

```diff
@@ -0,0 +1,11 @@
+## Notes
```

Comment:
We recently started using github releases for release notes https://github.com/scalameta/scalameta/releases I've found this to be very convenient

Reply:
So we can remove this? Fine by me. One less document to maintain.

Reply:
Yeah I'd remove it, having release notes in github releases lower the barrier to update/polish the notes which is a good thing IMO. No need for a PR

Reply:
:+1: